website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -121,11 +115,19 @@ resource "azurerm_virtual_network" "test_advanced_network" {
 resource "azurerm_subnet" "test_subnet" {
   name                      = "akc-1-subnet"
   resource_group_name       = "${azurerm_resource_group.test.name}"
-  network_security_group_id = "${azurerm_network_security_group.test_advanced_network.id}"
   address_prefix            = "10.1.0.0/24"
   virtual_network_name      = "${azurerm_virtual_network.test_advanced_network.name}"
 }
 
+
+resource "azurerm_log_analytics_workspace" "test" {
```

Comment:
Is there a reason this resource was added 🤔

Reply:
yeah, it's referenced below but not present